static/js/publisher/release/components/releasesConfirmDetails/index.js

```diff
@@ -1,16 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
 
+import {
+  updateProgressiveReleasePercentage,
+  setTemporaryProgressiveReleaseKeys,
+  removeTemporaryProgressiveReleaseKeys
+} from "../../actions/pendingReleases";
 import { isProgressiveReleaseEnabled } from "../../selectors";
 
 import progressiveTypes from "./types";
 import ReleaseRow from "./releaseRow";
 import CancelProgressiveRow from "./cancelProgressiveRow";
 import ProgressiveRow from "./progressiveRow";
+import ProgressiveRowGroup from "./progressiveRowGroup";
 import CloseChannelsRow from "./closeChannelsRow";
 
-const ReleasesConfirmDetails = ({ updates, isProgressiveReleaseEnabled }) => {
+const ReleasesConfirmDetails = ({
+  updates,
+  isProgressiveReleaseEnabled,
+  updateProgressiveReleasePercentage,
+  setTemporaryProgressiveReleaseKeys,
+  removeTemporaryProgressiveReleaseKeys
+}) => {
+  const [useGlobal, setGlobal] = useState(true);
```

Comment:
`useGlobal` is using hooks naming convention of `use*` which may be confusing when reading the code and may cause problems with linting tools (if we start using some hooks features in linting).

Reply:
Why would it cause problems with linting? Do you have a link I could have a gander at?

Reply:
I'm not sure exactly that it may be a problem, but I think the `use*` name convention is used to determine if given function is a hook or not: https://reactjs.org/docs/hooks-rules.html

Maybe it will not be an issue if it's not a function, but still may be confusing to read in the code.